packages/scripts/README.md

```diff
@@ -528,7 +528,7 @@ import './style.css';
 When you run the build using the default command `wp-scripts build` (also applies to `start`) in addition to the JavaScript file `index.js` generated in the `build` folder, you should see two more files:
 
 1. `index.css` – all imported CSS files are bundled into one chunk named after the entry point, which defaults to `index.js`, and thus the file created becomes `index.css`. This is for styles used only in the editor.
-2. `style.css` – imported `style.css` file(s) (applies to SASS and SCSS extensions) get bundled into one `style.css` file that is meant to be used both on the front-end and in the editor.
+2. `style-index.css` – imported `*style.css` file(s) (applies to SASS and SCSS extensions) get bundled into one `style-index.css` file that is meant to be used both on the front-end and in the editor.
```

Comment:
Should we actually limit it to only `style.css` files?

Reply:
I guess it's fine to limit it. It's a breaking change regardless that I still need to include in the CHANGELOG file. Previously, we would bundle all `style` files into one even when there were multiple entry points. I found a way to resolve it.

Reply:
Well, it's an interesting case because it really never worked as expected 🤣 

We can classify as a bug fix and adjust however we want. I will test more strict regexp on Monday.

Reply:
> Previously, we would bundle all `style` files into one even when there were multiple entry points.

Right, now we only have one per entry point which is good IMO. My comment was more about that also files like `foostyle.css` are handled by this rule which I think we should prevent.